components/welcome/actions/update-event/update-event.mjs

```diff
@@ -0,0 +1,95 @@
+import utils from "../../common/utils.mjs";
+import app from "../../welcome.app.mjs";
+
+export default {
+  key: "welcome-update-event",
+  name: "Update an Existing Event",
+  description: "Updates an existing event. [See the documentation](https://app.experiencewelcome.com/api-docs/index.html)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    eventId: {
+      propDefinition: [
+        app,
+        "eventId",
+      ],
+    },
+    name: {
+      propDefinition: [
+        app,
+        "name",
+      ],
+    },
+    description: {
+      propDefinition: [
+        app,
+        "description",
+      ],
+    },
+    startTime: {
+      propDefinition: [
+        app,
+        "startTime",
+      ],
+    },
+    endTime: {
+      propDefinition: [
+        app,
+        "endTime",
+      ],
+    },
```

Comment:
```suggestion
    startTime: {
      propDefinition: [
        app,
        "startTime",
      ],
      optional: true,
    },
    endTime: {
      propDefinition: [
        app,
        "endTime",
      ],
      optional: true,
    },
```

Reply:
Maybe I'm not looking at the right [docs](https://app.experiencewelcome.com/api-docs/index.html)? From what I'm seeing, they're required for creating an event but not updating (PATCH).
<img width="386" alt="Screenshot 2024-08-26 at 12 38 29 PM" src="https://github.com/user-attachments/assets/f92c69af-cd98-4141-8e33-5a93d49702ba">